blobfuse/src/DfsProperties.cpp

```diff
@@ -99,7 +99,8 @@ dfs_properties adls_client_ext::get_dfs_path_properties(const std::string &files
         props.owner = http->get_response_header(constants::header_ms_owner);
         props.group = http->get_response_header(constants::header_ms_group);
         props.permissions = http->get_response_header(constants::header_ms_permissions);
-        props.acl = http->get_response_header(constants::header_ms_acl);
```

Comment:
This gives file/dir permission related info in case of dfs endpoint.

Reply:
acl was not returned. Only permissions. So I removed.